es-app/src/SystemData.cpp

```diff
@@ -210,7 +210,9 @@ SystemData* SystemData::loadSystem(pugi::xml_node system)
 
 		// if there appears to be an actual platform ID supplied but it didn't match the list, warn
 		if (str != NULL && str[0] != '\0' && platformId == PlatformIds::PLATFORM_UNKNOWN)
+		{
```

Comment:
Added braces around LOG macro to resolve warning below.

/home/devel/temp/EmulationStation_Stable/es-app/src/SystemData.cpp:214:3: warning: add explicit brace
s to avoid dangling else [-Wdangling-else]
                else if (platformId != PlatformIds::PLATFORM_UNKNOWN)
                ^